.circleci/config.yml

```diff
@@ -1214,109 +1309,26 @@ workflows:
-      - run-test-ios-15:
-          xcode_version: '14.3.0'
-          install_swiftlint: false
-      - run-test-watchos
-      - run-test-tvos
-      # Pre-iOS 15 requires macOS 12 which requires Xcode 14.2
-      # See https://circleci.com/docs/using-macos/#supported-xcode-versions
-      - run-test-ios-14:
-          xcode_version: '14.2.0'
-          install_swiftlint: false
-      - run-test-ios-13:
-          xcode_version: '14.2.0'
-          install_swiftlint: false
-          <<: *release-branches-and-main
-      - build-tv-watch-and-macos
-      - build-visionos
-      - backend-integration-tests-SK1:
-          filters:
-            branches:
-              # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
-              ignore: /pull\/[0-9]+/
-      - backend-integration-tests-SK2:
-          filters:
-            branches:
-              # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
-              ignore: /pull\/[0-9]+/
-      - backend-integration-tests-other:
-          filters:
-            branches:
-              # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
-              ignore: /pull\/[0-9]+/
-      - backend-integration-tests-offline:
-          # These tests are flaky due to FB13133387 so we don't want the noise in every PR
-          <<: *release-branches-and-main
-      - backend-integration-tests-custom-entitlements:
-          filters:
-            branches:
-              # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
-              ignore: /pull\/[0-9]+/
+      - run-test-ios-17
+      - pod-lib-lint
 
   deploy:
     when:
-      not:
-        equal: [scheduled_pipeline, << pipeline.trigger_source >>]
+      and:
+        - not:
+            equal: [scheduled_pipeline, << pipeline.trigger_source >>]
+        - matches: 
+            pattern: "^release/.*$"
+            value: << pipeline.git.branch >>
     jobs:
-      - release-checks:
-          <<: *release-branches
-      - installation-tests-cocoapods:
-          <<: *release-branches
-      - installation-tests-swift-package-manager:
-          <<: *release-branches
-      - installation-tests-custom-entitlement-computation-swift-package-manager:
-          <<: *release-branches
-      - installation-tests-receipt-parser:
-          <<: *release-branches
-      - installation-tests-carthage:
-          <<: *release-branches
-      - installation-tests-xcode-direct-integration:
-          <<: *release-branches
+      - release-checks
       - hold:
           type: approval
           requires:
             - release-checks
-            - installation-tests-cocoapods
-            - installation-tests-swift-package-manager
-            - installation-tests-custom-entitlement-computation-swift-package-manager
-            - installation-tests-carthage
-            - installation-tests-xcode-direct-integration
-            - installation-tests-receipt-parser
-          <<: *release-branches
```

Comment:
Deploy now only requires `release-checks`, but all of the tests will trigger anyway. 
I debated with myself on whether to require all the jobs anyway, but given how flaky they've been, I think it might not be a bad idea to let us approve and deploy even if a test isn't passing, and leave it up to reviewer discretion. 

Before we were only requiring installation tests to pass, so it's not _that_ different now